### ISSUE TITLE
How to manipulate or interpolate images?

### ISSUE THREAD

**Issue by TakeruShiraishi** [ThomasMrY/DisDiff#6]
Thank you for your great work! I have a question about your work.
How to manipulate or interpolate images like previous works, such as diffusion autoencoders or PDAE?

**Comment by ThomasMrY** [comment]
Thank you for interesting at our work. We are happy to address any problem you have. Please feel free to contact us.
You can see line 1474-1485 in ldm/models/diffusion/ddpm.py
`if plot_swapped_concepts:
            x_samples_list = []
            with self.ema_scope("Plotting Swapping"):
                for cdx in range(self.model.diffusion_model.latent_unit):
                    swapped_c = c.clone()
                    swapped_c = torch.stack(swapped_c.chunk(self.model.diffusion_model.latent_unit, dim=1), dim=1)
                    swapped_c[:,cdx] = swapped_c[0,cdx][None,:].repeat(c.shape[0],1)
                    samples, z_denoise_row = self.sample_log(cond=swapped_c.reshape(c.shape[0],-1),batch_size=N,ddim=use_ddim,
                                                            ddim_steps=ddim_steps,eta=ddim_eta)
                    x_samples = self.decode_first_stage(samples)
                    x_samples_list.append(x_samples)
                log["samples_swapping"] = torch.cat(x_samples_list, dim=0)`
We manipulate images by swapping the representations.
